src/main/java/com/yurjinia/common/security/jwt/service/JwtService.java

```diff
@@ -60,6 +61,19 @@ public void blacklistToken(String token, long expirationTime) {
         jwtBlacklistService.blacklistToken(token, expirationTime);
     }
 
+    public void blacklistTokenIfExists(HttpServletRequest httpServletRequest) {
```

Comment:
is there any difference between this method and `jwtBlacklistService.blacklistToken()`?